testing-guidance/workflow-tests/physics/physics-lkg-workflows.md

```diff
@@ -110,8 +110,8 @@ Any project that has the following Gems enabled:
+| Force Region Types                        | <ol><li>Create an entity with a Force Region Component for each [PhysX Force Region Force Type](#PhysX-Force-Region-Force-Types) under test and configure it for the PhysX Force Regions under test using a default PhysX Primitive Collider as the collider dependency.<li>Modify the Force Region's Force Type properties. (**EG:** Setting a Magnitude and X,Y,Z directions for World Space)<ul><li>Test further by testing the boundaries defined in the docs.</ul></li><li>Set up the scene so that an entity will collide with the Force Region under test.</li><li>Run the Simulation in Editor Game Mode or in the Game Launcher.</li></ol> | <ul><li>Force Region Can be applied to an entity.</li><li>If dependent components are missing, the Force Region component notifies the user and gives them a resolution path.</li><li>Force Region types can be added to a force region and their properties can be modified.</li><li>Entities that collide with the Force Region will react according to the the expected Force applied.</li><ul><li>Force Regions apply forces in all vectors.</li><li>Force Region Net Forces are as expected.</li><li>Force Regions do not cause entities to quiver.</li></ul></ul> |
+| Force Region Shapes                       | <ol><li>Rerun above scenario but swap out default PhysX Primitive Collider component to support other [PhysX Primitive Collider shapes](#physx-collider-shapes), PhysX Mesh Collider or [PhysX Shape Collider](#PhysX-Shape-Collider-shapes) under test.</li><li>Add the desired [PhysX Force Region Force Type(s)](#PhysX-Force-Region-Force-Types) under test.</li><li>Set up the scene so that an entity will collide with the Force Region under test.</li><li>Run the Simulation in Editor Game Mode or in the Game Launcher.</li></ol>                                                                                                                                   | <ul><li>Force Region Can be applied to Entity with desired PhysX Primitive Collider shape, PhysX Mesh Collider or PhysX Shape Collider w/ desired shape.</li><li>Entities that collide with the Force Region will react according to the the expected Force applied.</li></ul>                                                                                                                                                                                                                                                                                                                         |
```

Comment:
```suggestion
| Force Region Shapes                       | <ol><li>Rerun above scenario but swap out default PhysX Primitive Collider component to support other [PhysX Primitive Collider shapes](#physx-primitive-collider-shapes), PhysX Mesh Collider or [PhysX Shape Collider](#PhysX-Shape-Collider-shapes) under test.</li><li>Add the desired [PhysX Force Region Force Type(s)](#PhysX-Force-Region-Force-Types) under test.</li><li>Set up the scene so that an entity will collide with the Force Region under test.</li><li>Run the Simulation in Editor Game Mode or in the Game Launcher.</li></ol>                                                                                                                                   | <ul><li>Force Region Can be applied to Entity with desired PhysX Primitive Collider shape, PhysX Mesh Collider or PhysX Shape Collider w/ desired shape.</li><li>Entities that collide with the Force Region will react according to the the expected Force applied.</li></ul>                                                                                                                                                                                                                                                                                                                         |
```